datajoint/expression.py

```diff
@@ -73,6 +75,17 @@ def heading(self):
         """a dj.Heading object, reflects the effects of the projection operator .proj"""
         return self._heading
 
+    @property
+    def top(self):
+        """a top object to form the ORDER BY, LIMIT, and OFFSET clauses"""
+        if self._top and self._top["order_by"]:
+            if isinstance(self._top["order_by"], str):
+                self._top["order_by"] = [self._top["order_by"]]
+            if "KEY" in self._top["order_by"]:
+                i = self._top["order_by"].index("KEY")
```

Comment:
compare how this is done in `fetch.Fetch.__call__` and replicate the same processing.

Reply:
check out `_flatten_attribute_list` in `fetch.py`.